suite/runsuite.cmake

```diff
@@ -550,14 +552,17 @@ if (ARCH_IS_X86 AND UNIX AND (a64_on_x86_only OR NOT arg_automated_ci))
     " OFF ${arg_package} "")
 endif ()
 
-if (ARCH_IS_X86 AND UNIX AND cross_riscvxx_linux_only)
-# Do riscv cross-compilation if cross_riscvxx_linux_only is on.
+if (ARCH_IS_X86 AND UNIX)
+  #TODO i#3544: Run tests under QEMU
```

Comment:
Style nit: Add space between the `#` and `TODO`: `# TODO i#3544: ...`

Reply:
Done.